src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html

```diff
@@ -0,0 +1,225 @@
+            <br> You can press the <span class="text-muted glyphicon glyphicon-resize-full"></span> button in the top-right corner to display the information in a lightbox for better readability.
+          </p>
+        </div>
+      </div>
+      <br/>
+    </div>
+  </div>
+  <!--Question-->
+  <div class="instr-help-qn1" *ngIf="show_arr[1]">
+    <p><button type="button" class="btn btn-outline-primary" (click)="isCollapsed1 = !isCollapsed1" [attr.aria-expanded]="!isCollapsed1" aria-controls="answer1">
+      How do I edit a student's details after enrolling the student?
+    </button></p>
+    <div id="answer1" [ngbCollapse]="!isCollapsed1">
+      <div class="card">
+        <div class="card-body">
+          <p>
+            To edit the name, section, team, contact email, or instructor comments of Student A from Course B:
+          </p>
+          <ol>
+            <li>
+              Go to the <b>Students</b> page and click the panel heading for Course B. You will see a list of students enrolled in Course B.
+            </li>
+            <li>
+              Click the
+              <button class="btn btn-default btn-xs">Edit</button> button in the last column of the row corresponding to Student A.
+              <br>
+            </li>
+            <li>
+              In the new page that opens, edit the relevant fields of Student A's details. The page will look similar to the example below.
+            </li>
+            <li>
+              Click
+              <button class="btn btn-primary btn-s">Save Changes</button> to save your changes to Student A's details.
+            </li>
+          </ol>
+          <p>
+            Note that moving a student to a different team (i.e. changing the student's Team ID) will change the student's team in all existing sessions in the course.
+          </p>
+        </div>
+      </div>
+      <br/>
+    </div>
+  </div>
+  <!--Question-->
+  <div class="instr-help-qn2" *ngIf="show_arr[2]">
+    <p>
+      <button type="button" class="btn btn-outline-primary" (click)="isCollapsed2 = !isCollapsed2" [attr.aria-expanded]="!isCollapsed2" aria-controls="answer2">
+      How do I view all the responses a student has given and received?
+      </button>
+    </p>
+    <div id="answer2" [ngbCollapse]="!isCollapsed2">
+      <div class="card">
+        <div class="card-body">
+          <p>
+            To view the responses that Student A from Course B has given and received:
+          </p>
+          <ol>
+            <li>
+              Go to the <b>Students</b> page and click the panel heading for Course B. You will see a list of students enrolled in the course.
+            </li>
+            <li>
+              Click
+              <button class="btn btn-xs btn-default">All Records</button> button corresponding to Student A to access all the responses Student A has given and received.
```

Comment:
cannot really see `btn-default` in the new bootstrap, can you try `btn-secondary` or `btn-light`?
also `btn-xs` changed to `btn-sm` in bootstrap 4.0

Reply:
Yes you are right. I have fixed all the buttons in the file - made them into `btn-secondary` and `btn-sm`